### PR TITLE
#144

### PR DESCRIPTION
**Pull request by DiliBau**

### CODE REVIEW

**Comment by radutopala** [comment]
@DiliBau pls clean up description texts from migrations and so forth.

File: backend/app/DoctrineMigrations/Version20200811110857.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 use Doctrine\DBAL\Schema\Schema;
 
 /**
- * Auto-generated Migration: Please modify to your needs!
+ * This migration was added by @maxlenash because he was dropped on the head when he was little.
```
Comment (radutopala) [inline backend/app/DoctrineMigrations/Version20200811110857.php:9]: Cleanup

File: backend/src/AppBundle/DataFixtures/Team/LoadProjectStatusData.php [hunk 2]
```diff
@@ -26,7 +26,20 @@ public function load(ObjectManager $manager)
         ];
 
         foreach ($data as $row) {
-            $ps = new ProjectStatus();
+            /**
+             * @maxlenash has added https://github.com/CamprGmbH/campr/blob/develop/backend/app/DoctrineMigrations/Version20200811110857.php
```
Comment (radutopala) [inline backend/src/AppBundle/DataFixtures/Team/LoadProjectStatusData.php:30]: Cleanup

File: backend/src/AppBundle/DataFixtures/Team/LoadProjectStatusData.php [hunk 3]
```diff
@@ -26,7 +26,20 @@ public function load(ObjectManager $manager)
...
+             *
+             * Good job moron!
```
Comment (radutopala) [inline backend/src/AppBundle/DataFixtures/Team/LoadProjectStatusData.php:32]: Cleanup

**Comment by DiliBau** [comment]
@radutopala cleanup done

**Review by radutopala — APPROVED** [review]

**Comment by radutopala** [comment]
@DiliBau pls deploy to QA first
@CAMPR-Manuel @cristobalcampr pls check QA after the deploy

**Comment by cristobalcampr** [comment]
Hi guys @DiliBau @radutopala , unfortunately I get an symfony error immediately after typing in the url **_qa.campr.cloud_**, see screenshot attached.

Bildschirmfoto vom 2020-11-13 14-29-02

**Pull request merged**